pkg/client/testdata/PrintPodStatus.json

```diff
@@ -0,0 +1,2818 @@
+{
```

Comment:
I'd really not prefer to have a 2k line test file here if we dont need it.

Lets just scope this to the ones that show our common use cases/errors.